hoomd/mpcd/update.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2009-2024 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+r"""Reverse nonequilibrium shear flow of an MPCD fluid.
+
+Reverse nonequilibirum shear flow of an MPCD fluid is established by imposing
+a known momentum flux on the system and measuring the resulting shear rate. This
+provides a way to calculate the shear viscosity of the solvent.
```

Comment:
This documentation should be for the module, not the class. It can be a short description that these are MPCD updaters.